routes/util.go

```diff
@@ -38,3 +39,14 @@ func param(r *http.Request, key string) string {
 func redir(url string) http.HandlerFunc {
 	return http.RedirectHandler(url, http.StatusPermanentRedirect).ServeHTTP
 }
+
```

Comment:
Ooo this is actually pretty close to my proposed addition to chi - https://github.com/go-chi/chi/issues/709

Can we get rid of `redir` with this? It's a superset right?

Reply:
Technically not quite if there's braces in `redir` 's argument. But yeah I can get rid of `redir`.